src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see
+ * an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString
+    .split(';')
+    .filter(el => el.indexOf(':') !== -1);
+  const finObj = {};
+
+  sourceArray.map(callback);
+
+  function callback(element) {
+    const clearElement = element
+      .replace(' ', '')
+      .replace('\n', '');
```

Comment:
what if there are more then  1 whitespace or `\n` - line break? there is a function to `trim` such values`